src/azul/__init__.py

```diff
@@ -906,9 +915,10 @@ def fail_queue_names(self) -> List[str]:
     @property
     def work_queue_names(self) -> List[str]:
         return [
-            queue_name(retry=retry)
-            for queue_name in (self.notifications_queue_name, self.tallies_queue_name)
-            for retry in (False, True)
+            self.redrive_queue_name(),
```

Comment:
Should the redrive queue be consider a worker queue? It's surely an intermediary.

Reply:
To answer this question, consider what code paths are enabled by designating a queue as a work queue. Determine if we want those code paths to be enabled for the redrive queue.

Reply:
I believe yes. From the permissions stand point, a work queue can modify and manipulate messages. This allows for the queue purging via the AzulClient, and the wait for queues to stabilize process following indexing.

Reply:
Right.

> From the permissions stand point, a work queue can modify and manipulate messages.

makes no sense though. Queues *contain* messages. They neither modify nor manipulate messages.